src/main/java/org/jabref/gui/JabRefMain.java

```diff
@@ -139,4 +149,25 @@ private static void configureProxy(ProxyPreferences proxyPreferences) {
             Authenticator.setDefault(new ProxyAuthenticator());
         }
     }
+
+    private static void clearOldSearchIndices() {
+        Path appData = Path.of(AppDirsFactory.getInstance().getUserDataDir(JabRefFrame.FRAME_TITLE, null, "org.jabref"));
+        Path currentIndexPath = Path.of(AppDirsFactory.getInstance().getUserDataDir(JabRefFrame.FRAME_TITLE, SearchFieldConstants.VERSION, "org.jabref"));
```

Comment:
This appears twice here in the PR. Can it be factored out to another method? Just use `jabref` as string, no `FRAME_TITLE`.